payment_redsys/models/redsys.py

```diff
@@ -218,9 +214,10 @@ def redsys_form_generate_values(self, values):
         )
         return redsys_values
 
-    @api.multi
     def redsys_get_form_action_url(self):
-        return self._get_redsys_urls(self.environment)["redsys_form_url"]
+        self.ensure_one()
+        environment = "prod" if self.state == "enabled" else "test"
```

Comment:
If disabled, can it be selected as payment method?

Reply:
Yes, but just can be used on test mode